### ISSUE TITLE
Error running gigasecond test

### ISSUE THREAD

**Issue by karptonite** [exercism/xhaskell#80]
I'm getting the following error when trying to run the Gigasecond test:
```
gigasecond_test.hs:9:15:
    Couldn't match expected type ‘time-1.5.0.1:Data.Time.Format.Locale.TimeLocale’
                with actual type ‘System.Locale.TimeLocale’
    NB: ‘time-1.5.0.1:Data.Time.Format.Locale.TimeLocale’
          is defined in ‘Data.Time.Format.Locale’ in package ‘time-1.5.0.1’
        ‘System.Locale.TimeLocale’
          is defined in ‘System.Locale’ in package ‘old-locale-1.0.0.7’
    In the first argument of ‘readTime’, namely ‘defaultTimeLocale’
    In the expression:
      readTime defaultTimeLocale (iso8601DateFormat (Just "%T%Z"))
```
I'm not sure if this is a problem in the test or if it is a problem in my Haskell install or even if it is in my solution (although from the error message, it doesn't look like the problem is in my solution). I'm (obviously) pretty new to Haskell, so I'm sorry I can't be more help in diagnosing this.

I'm on OSX with ghc, and installed via Brew, and HUnit installed via cabal, apparently up to date on everything. 

Apologies if this is my problem, and should have been posted elsewhere!

**Comment by kytrinyx** [comment]
This is the right place for it. @etrepum do you know what might be going on here?

**Comment by etrepum** [comment]
Pretty sure this is a problem with your solution, but I can't quite tell because you didn't post any of it.

**Comment by etrepum** [comment]
Also, which version of GHC are you using?

**Comment by karptonite** [comment]
GHC version 7.10.1
my solution:
```
module Gigasecond (fromDay)
where
import Data.Time.Clock
  
fromDay :: UTCTime -> UTCTime
fromDay = addUTCTime 1000000000
```

**Comment by karptonite** [comment]
I just checked out this repo, and ran the _test tests. Gigasecond is one of the ones that fails. Perhaps this is a Haskell version problem?

**Comment by etrepum** [comment]
Yeah, it was GHC 7.10. Your solution is fine. Thanks! :+1:

**Comment by etrepum** [comment]
It's a bit tricky to have this stuff all work without warnings from GHC 7.6 onwards! :)

**Comment by karptonite** [comment]
Thanks--that was fast! How long before the new version should be available with `exercism fetch`?

**Comment by etrepum** [comment]
Not sure, I don't have visibility beyond just maintaining the track. @kytrinyx ?

**Comment by karptonite** [comment]
Might be worth adding newer versions of Haskell to the .travis.yml file. I started on a pull request for it, but was unable to run the tests in Travis because the bootstrap.sh file use sudo.

**Comment by etrepum** [comment]
Lots of people still use 7.6 due to Linux distros, so we definitely need to keep that. At the time I set this up, Travis support for Haskell didn't really work, so I had to roll my own. Not sure if any of that has changed.

**Comment by kytrinyx** [comment]
I try to deploy daily-ish, but sometimes things get hectic and I miss a day or two. I'm deploying now.

**Comment by karptonite** [comment]
@etrepum it looks as if travis supports Haskell now. http://docs.travis-ci.com/user/languages/haskell/

**Comment by etrepum** [comment]
It's a bit more complicated than just using that. Best not to discuss this on a closed ticket, I created #82 for further discussion and to track any progress.
